pkgs/tools/admin/google-cloud-sdk/default.nix

```
@@ -21,7 +21,20 @@
 }:
 
 let
-  pythonEnv = python.withPackages (
+  python' = python.override {
```

```suggestion
  python = python3.override {
```
Is there a reason to use `python`?